### ISSUE TITLE
Depfu Badge Says Invalid

### ISSUE THREAD

**Issue by arch-lord** [depfu/feedback#21]
**TL;DR: There's some bug with Depfu's shields api at `https://depfu.com/github/shields/<user>/<repo>` that displays some repositories as invalid.**

I setup Depfu for my project [Blaggy](https://depfu.com/repos/arch-lord/blaggy). I wanted to put a depfu badge in my README.md, so I went to [Shields.io](http://shields.io/) and entered the correct information. The badge looks like this:
![image](https://user-images.githubusercontent.com/42556441/45058821-d7984200-b067-11e8-996c-979c17742ccc.png)

Thinking this might be a problem with Shields.io I checked Depfu's shield API at [this URL](https://depfu.com/github/shields/arch-lord/blaggy), but it, too, says "invalid":
![image](https://user-images.githubusercontent.com/42556441/45058895-0e6e5800-b068-11e8-90fe-e1eec76e973e.png)

The last thing is that the badge on [my page at Depfu](https://depfu.com/repos/arch-lord/blaggy) works properly:
![image](https://user-images.githubusercontent.com/42556441/45058933-26de7280-b068-11e8-80ef-6164e16c70da.png)

Before you ask me why I can't just use that, it's because I want a style only available (as far as I know) on Shields.io.

**Comment by arch-lord** [comment]
Well, it's fixed now after about 5 days.

**Comment by theflow** [comment]
Sorry for the problems, looks like you've disabled and enabled the repo earlier today and that fixed it.

**Comment by arch-lord** [comment]
Yes. I believe that was the solution.

Sent from Mail for Windows 10
